### DISCUSSION TITLE
Dagster Daemon Logs when schedule/sensors are turned off

### DISCUSSION THREAD

**Discussion by sivasankarreddyy** [dagster-io/dagster#21182] [Q&A]
Hi All,

I am working on setting up alerts when any schedules/sensors are turned off on Dagster UI. 

Release notes of 1.6.9 says, we have this feature enabled already : https://docs.dagster.io/changelog#:~:text=Log%20messages%20in%20the%20Dagster%20daemon%20for%20unloadable%20sensors%20and%20schedules%20have%20been%20removed.

But I don't find any logs as such in dagster daemon when I turn off schedules. Can you please help me if I am misconfiguring anything. 

My current setup 

Dagster webserver version : 1.6.11
Dagster Daemon : 1.6.11
Both are using same workspace yaml file. 

Thank you
Siva

**Comment by garethbrickman** [comment]
I think you're misinterpreting that release note- it is referring to _removing_ logging in the Dagster daemon for when schedules/sensors are unloadable. 

I don't think the Dagster UI emits a log for when a user enables or disables a schedule or sensor. I know that the[ GraphQL API](https://docs.dagster.io/concepts/webserver/graphql) can be queried to check the current status of the schedule/sensor but this doesn't include by whom it was enabled or disabled and when that occurred.
